### ISSUE TITLE
Less noisy logs?

### ISSUE THREAD

**Issue by AndyRPH** [simeononsecurity/docker-rtklib-onocoy-rtkdirect#5]
Hi,

My instance of this drops onocoy multiple times a day but not a specific interval, and usually, but not always, the rtkdirect dashboard still shows it as online.   

Is there an easy way to make the onocoy logs more verbose and the rtkdirect logs less? There are multiple lines per second for rtkdirect that make the logs hard to parse.

**Comment by simeononsecurity** [comment]
> Hi,
> 
> My instance of this drops onocoy multiple times a day but not a specific interval, and usually, but not always, the rtkdirect dashboard still shows it as online.
> 
> Is there an easy way to make the onocoy logs more verbose and the rtkdirect logs less? There are multiple lines per second for rtkdirect that make the logs hard to parse.

Ntripserver doesn't have any control of verbosity.
For rtklib, you can simply download the repo and modify the instances where the `$RTKLIB_VERBOSITY` variable is called to change the verbosity of the different instances of it.
https://github.com/simeononsecurity/docker-rtklib-onocoy-rtkdirect/blob/d4f5cd261b1ca04b5e1c30b526f996a316177e22/docker-init.sh#L121C131-L121C148

Then, you'd just follow the typical docker build and run commands for locally built images

If you don't know how to do that, you can learn with other examples here
https://docs.docker.com/get-started/workshop/02_our_app/
